test/ingredientRepo-test.js

```diff
@@ -0,0 +1,61 @@
+const chai = require('chai');
+const expect = chai.expect;
+const Ingredient = require('../src/Ingredient');
+const IngredientRepo = require('../src/IngredientRepo');
+
+describe('Ingredient Repo', () => {
+  let ingredientList;
+  let defaultIngredientList;
```

Comment:
refactor line 7 to say `let ingredientList, defaultIngredientList, ingredientData;`